versions.txt

```diff
@@ -7,10 +7,10 @@ docker_ubuntu_version=16.04
 docker_engine_fedora_version=1.12.1
 docker_engine_ubuntu_version=1.12.1-0~xenial
 clear_vm_image_version=14700
-clear_vm_kernel_version=4.5-57
+clear_vm_kernel_version=4.9.4-57
```

Comment:
Note, I have recently asked @miguelinux if/when/should we update the container kernel to the latest longerm (4.9.24)... I think that is now overdue.